views/halloffame.html

```diff
@@ -69,6 +83,81 @@
 			</section >
 			<section class="hof-stats right-side">
 				<h2>My Stats</h2>
+				<div class="stats-helper">
+					<p><span class="regNum"></span> regular HOF conversations</p>
+					<p><span class="gpNum"></span> gold plated conversations</p>
+					<p><span class="gNum"></span> golden conversations</p>
+				</div>
+				<div id="chart"></div>
+			    <script>
```

Comment:
Since this script is 50+ lines of code, it might be a good idea to move it into its own .js file, so as to separate it from the HTML and dedicate a place where we can add to it in the future.